graphql_test.go

```diff
@@ -370,9 +370,7 @@ func TestErrorWithExtensions(t *testing.T) {
 				}
 			`,
 			ExpectedResult: `
-				{
-					"FindDroid": null
-				}
+				null
```

Comment:
It would be good to have a test which ensures the behavior is as described in the specification: https://facebook.github.io/graphql/draft/#sec-Errors-and-Non-Nullability

Specifically, we should test that children of non-null nodes are not returned, and that the siblings of the non-null node are also not returned (because the error propagates all the way up to the nearest nullable parent.

Reply:
on unrelated note. Great to see you back Tony :-)